src/compiler/emitter.ts

```diff
@@ -1343,6 +1343,18 @@ const enum PipelinePhase {
     Emit,
 }
 
+/** @internal */
+export const createPrinterWithDefaults = memoize(() => createPrinter({}));
+
+/** @internal */
+export const createPrinterWithRemoveComments = memoize(() => createPrinter({ removeComments: true }));
+
+/** @internal */
+export const createPrinterWithRemoveCommentsNeverAsciiEscape = memoize(() => createPrinter({ removeComments: true, neverAsciiEscape: true }));
+
+/** @internal */
+export const createPrinterWithRemoveCommentsOmitTrailingSemicolon = memoize(() => createPrinter({ removeComments: true, omitTrailingSemicolon: true }));

```

Comment:
I'm lazy so instead of coming up with a memoize + cache key, I'm just hardcoding these 4 which are all of the easy ones.

Reply:
Now that this isn't a draft (seems to actually be beneficial!), I'll have to think of something a little better, maybe.

Reply:
Why? What's the problem with this?

Reply:
It doesn't scale up or accept non-boolean options; there are a few leftover in services which pass in more complicated (but still limited) number of options that I can't easily emulate like this.